shared/src/main/scala/squants/information/Information.scala

```diff
@@ -154,6 +175,91 @@ object Yobibytes extends InformationUnit {
   def symbol = "YiB"
 }
 
+object Bits extends InformationUnit {
+  def conversionFactor = 0.125d
```

Comment:
Maybe this should be more explicitly set to `1/8`

Reply:
This can be either `0.125d` or `1d/8` (with double literal or other floating point marking) as both will result in the same value. Originally I've used literal form as it is cleaner in my opinion.